src/Features/ActivityPanels.php

```diff
@@ -37,7 +37,7 @@ public static function get_instance() {
 	 */
 	public function __construct() {
 		add_filter( 'wc_admin_get_user_data_fields', array( $this, 'add_user_data_fields' ) );
-		add_action( 'woocommerce_components_settings', array( $this, 'component_settings' ), 20 ); // Run after Automattic\WooCommerce\Admin\Loader.
+		add_action( 'woocommerce_shared_settings', array( $this, 'component_settings' ), 20 ); // Run after Automattic\WooCommerce\Admin\Loader.
```

Comment:
I know it wasn't introduced in this commit, but should this be `add_filter()` instead?

Reply:
Looks like it...

Reply:
Fixed in the alternative pull I did (https://github.com/woocommerce/woocommerce-admin/pull/2917)